cmd/es-rollover/app/index_options_test.go

```diff
@@ -61,7 +61,7 @@ func TestRolloverIndices(t *testing.T) {
 			expected: []expectedValues{
 				{
 					prefix:               "mytenant-jaeger-span-archive",
-					templateName:         "mytenant-jaeger-span",
```

Comment:
Why is this changing? Shouldn't the template name keep the prefix name? 

The new golang implementation has to be backwards compatible.

Reply:
I shouldn't 

https://github.com/jaegertracing/jaeger/pull/3258/files#diff-59567de3853ac61d1a7460823020ee5c2fe7680353df87011f4cd1df1b132f45L83

Look as this calls, the template has no prefix.  It was my mistake to include the prefix in the first PR for this.

Reply:
wait. I see one mistake , don't merge this please.

Reply:
Fixed, you're right, the template name should be prefixed, the mapping name used to build the template should not.